km_api/know_me/admin.py

```diff
@@ -130,10 +130,11 @@ class ProfileAccessorAdmin(admin.ModelAdmin):
     Admin for the ``ProfileAccessor`` model.
     """
     fields = ('km_user_accessor', 'profile', 'can_write')
-    list_display = ('km_user_accessor', 'profile', 'can_write')
+    list_display = ('profile', 'can_write')
     search_fields = (
-        'km_user_accessor__user__first_name',
-        'km_user_accessor__user__last_name')
+        'profile__name',
+        'km_user_accessor__user_with_access__first_name',
+        'km_user_accessor__user_with_access__last_name')
```

Comment:
Are these correct? Is there any way for me to test these in the future so as to not accidentally add things that don't exist (as I did in the previous commit)?